Automattic/jetpack
# Update/hide customizer and custom css on block based themes

Fixes https://github.com/Automattic/wp-calypso/issues/71130

## Proposed changes:

* We remove the `Customize` and `Additional CSS` submenus when the theme is block-based.
* We disable the Safe CSS free trial since it is returning a Custom CSS comment that makes the Additional CSS submenu visible.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sandbox public API.
* Copy the branch in WPCom and use a block based theme.
* The `Customize` and `Additional CSS` submenus should be hidden.
* Switch to a non-block based theme.
* The submenus should now be visible.
* Use this branch in an Atomic site and repeat the steps.
* Use this branch in a JN site and make sure that it works just as it did before applying this branch.
